lib/utils.js

```diff
@@ -188,116 +109,25 @@ function isStandardBrowserEnv () {
   )
 }
 
-/**
- * Iterate over an Array or an Object invoking a function for each item.
- *
- * If `obj` is an Array callback will be called passing
- * the value, index, and complete array for each item.
- *
- * If 'obj' is an Object callback will be called passing
- * the value, key, and complete object for each property.
- *
- * @param {Object|Array} obj The object to iterate
- * @param {Function} fn The callback to invoke for each item
- */
-function forEach (obj, fn) {
```

Comment:
https://lodash.com/docs/4.17.4#forEach

if keys with `length` lodash forEach has different behavior with this custom forEach method

Reply:
Thanks, that's part of the upcoming PRs described in #1333, where we update the usage of `forEach` to `forOwn` on two or three occasions that cause this trouble.

The problem here was that we implemented all these changes and only then sat down an split it to logical units, so some small changes might have landed in a different step than anticipated.